src/taipy/core/scenario/_scenario_manager.py

```diff
@@ -206,25 +208,27 @@ def _delete(cls, scenario_id: ScenarioId):  # type: ignore
         scenario = cls._get(scenario_id)
         if scenario.is_primary:
             if len(cls._get_all_by_cycle(scenario.cycle)) > 1:
-                raise DeletingPrimaryScenario
+                raise DeletingPrimaryScenario(
+                    f"Scenario {scenario.id}, which has config id {scenario.config_id}, is primary and there are other scenarios in the same cycle."
```

Comment:
Here, there is a point. Otherwise, there are no points.
@FabienLelaquais, do you use points in this case?

Reply:
We want periods (dots) at the end of sentences. That would be English rules.